### ISSUE TITLE
bug: body hx-boost = true + form tag = Uncaught Type Error: path is null

### ISSUE THREAD

**Issue by wbcat** [bigskysoftware/htmx#3068]
using htmx 2.0.3 the following html document

```html
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<title>Title</title>
	<script src="htmx.js"></script>
</head>
<body hx-boost="true">
	<form>
	</form>
</body>
</html>
```

leads to 
`Uncaught TypeError: path is null`
in
`htmx.js:2326:13`

htmx 2.0.2 is fine

**Comment by kkpo4553** [comment]
Hello, we tried to solve the issue.

This is what we did:

Added a null check for the path variable before accessing its properties to prevent the 'Uncaught TypeError: path is null' error.

You can review changes in this commit: https://github.com/kkpo4553/bigskysoftware-htmx-3068/commit/84532718238f96a9d1aac9a796225c6fe9019bd1.


> [!CAUTION]
> Disclaimer: The concept of solution was created by AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---

If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.

**Comment by MichaelWest22** [comment]
This issue has already been fixed in the latest dev build here #2967 so fix will be in the next release

**Comment by wbcat** [comment]
Thank you!
